src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -374,6 +374,22 @@ def is_multi_npu():
     return (num_npus > 1)
 
 
+def is_voq_supervisor():
```

Comment:
Do we need this check to be specific to voq chassis ? Can we generalize to all chassis flavors

Reply:
I agree, can we make it `is_supervisor()` instead ?

Reply:
Yes. I'll change this to is_supervisor()

Reply:
Fixed.